### PR TITLE
Fix/mocha reporter

### LINKED ISSUE #30: clicking test case "arrow button" takes user nowhere
#### Issue Description

These buttons take the user to a blank page within codepen. To go back you'd have to right click within the preview window, rather than use the browsers nav buttons, because then you'd be going back not in the preview but for the whole page:

image

### PR DESCRIPTION
**Pull request by tbushman**
#### Pre-Submission Checklist

- [ ] Your pull request targets the `master` branch of freeCodeCamp/testable-projects-fcc
- [ ] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [ ] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [ ] Your changes have been tested either locally or using a newly created CDN based on your fork's testable-projects-fcc/build/bundle.js file
- [ ] Working changes have been added to the build by running `npm run build`

#### Type of Change

- [ ] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [ ] Tested changes locally.

- [ ] Closes currently open issue: Closes #30 

#### Description

Customized mocha reporter including markdown support using the javascript calculator project as a demonstration / test case. **Don't merge yet.**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -169,6 +194,81 @@ export function FCCResetTests(suite) {
...
+      }
+      console.log(err.stack)
+      // uncaught
+      if (err.uncaught) {
+        msg = 'Uncaught ' + msg;
+      }
+    }
+    var errStr = '' + msg +' \n '+ stack.split('\n').join('```  \n```') + '';
+    var fullTitle = test.parent.fullTitle() + ' ' + test.title;
+    str = ' <p class="fcc_external"><span><a target="_blank" title="run single test" href="/?grep='+encodeURIComponent(fullTitle)+'"> ' + test.title +  (test.speed !== 'fast' ? ' `duration: '+test.duration+'ms`' : '') +'</a></span><p class="fcc_err">'+errStr+'</p></p>';
```
Comment (tbushman) [inline src/index.js]: look into https://stackoverflow.com/a/11971032/3530394

File: src/project-tests/calculator-tests.js [hunk 2]
```diff
@@ -40,14 +41,20 @@ export default function createCalculatorTests() {
...
+    
+    describe('#Technology Stack', function() {
+      it(frontEndLibrariesStack, function() {
+        return true;
+      });
+    });
+    
     describe('#Tests', function() {
-      it(`1. My calculator should contain a clickable element containing an =
+      it(`My calculator should contain a clickable element containing an \`=\`
```
[the remark this answers is not in the document]
Comment (tchaffee) [reply src/project-tests/calculator-tests.js]: Well eliminating numbering would be cool. Does it re-start numbering in each section?

[the remark this answers is not in the document]
Comment (tbushman) [reply src/project-tests/calculator-tests.js]: Yes.

**Comment by tbushman** [comment]
Closing as work-in-progress

**Pull request closed without merging**